src/sentry/runner/decorators.py

```diff
@@ -25,3 +34,22 @@ def inner(ctx, *args, **kwargs):
             configure()
         return ctx.invoke(f, *args, **kwargs)
     return update_wrapper(inner, f)
+
+
+def log_level_option(f):
+    "Give ability to configure global logging level. Must be used before configuration."
+    import click
+    from functools import update_wrapper
+
+    @click.pass_context
+    @click.option('--loglevel', '-l', default=None,
+        help='Global logging level. Use wisely.',
+        envvar='SENTRY_LOG_LEVEL',
+        type=CaseInsensitiveChoice(LOG_LEVELS))
+    def inner(ctx, *args, **kwargs):
+        level = kwargs.pop('loglevel', None)
```

Comment:
I'd be ok with this if we gave it a value that clearly defines a no-op. In our set up, we would actually never set the global logging level, and it would be a bad idea to force that on others.